docs/make.jl

```diff
@@ -1,3 +1,36 @@
+#
+# Usage: from the package root directory, run
+#
+#    julia --proj=docs docs/make.jl
+#
+# To fix doctests instead of merely running them, use
+#
+#   DOCTEST=fix julia --proj=docs docs/make.jl
+#
+# To disable doctests, run
+#
+#   DOCTEST=off julia --proj=docs docs/make.jl
+#
+using Pkg
+Pkg.update()
+
+# parse some options
+doctest_arg = true
+if "--fix" in ARGS
+  doctest_arg = :fix
+else
+  tmp = get(ENV, "DOCTEST", "true")
```

Comment:
I think it is overkill to use both an env variable and and a command line argument to turn on "fix" mode. Thing is, I started with the idea of adding a command line argument, and just `--fix`, but then realized I also want to be able to turn off the doctests entirely. And then I got a bit lazy, as I'd have to decide what the args should be named (`--doctest=on/off/fix`?) and how to parse it ;-).

I think overall I'd find a command line argument nicer, and would be willing to implement it properly, but before I spend effort on it, I'd like to find out what others think.